BendAssist.App/BendAssists/MakeFlange.cs

```diff
@@ -1,15 +1,48 @@
-﻿namespace BendAssist.App.BendAssists;
+﻿using BendAssist.App.Model;
+
+namespace BendAssist.App.BendAssists;
 
 #region class MakeFlange --------------------------------------------------------------------------
 public sealed class MakeFlange : BendAssist {
-   #region Constructors ---------------------------------------------
-   #endregion
+    #region Constructors --------------------------------------------
+    public MakeFlange (Part part, PLine pline, double angle, double height, double radius) =>
+        (mPart, mPline, mAngle, mHeight, mRadius) = (part, pline, angle, height, radius);
+    #endregion
+
+    #region Methods -------------------------------------------------
+    public override bool Assisted () { return mCanAssist; }
 
-   #region Properties -----------------------------------------------
-   #endregion
+    /// <summary>Adds a new flange with the specified height, angle and radius.</summary>
+    /// Translates the selected pline to the given height
+    /// and creates a bendline with the given angle.
+    public override void Execute () {
+        var startPt = mPline.StartPoint; var endPt = mPline.EndPoint;

```

Comment:
This can be simplified by var (start, end) = (mPline.StartPoint, mPline.EndPoint);